examples/shadow_thing_sync_callbacks.ino

```diff
@@ -0,0 +1,103 @@
+#include "arduino_secrets.h"
+
+<<<<<<< HEAD
```

Comment:
It appears that there are merge conflict artifacts left within the whole file - please resolve.
```suggestion
```
Overall I appreciate the checking in of an example. Regardless - this example requires the **ArduinoIoTCloud** and therefore I suggest moving the example over to the ArduinoIoTCloud library.

Reply:
It was an old example that I put here erroneously.